pkg/events/server/nats.go

```diff
@@ -46,10 +47,11 @@ func NewNatsStream(opts ...nats.Option) (events.Stream, error) {
 	b := backoff.NewExponentialBackOff()
 	var stream events.Stream
 	o := func() error {
+		n := b.NextBackOff()
 		s, err := nats.NewStream(opts...)
-		if err != nil {
+		if err != nil && n > time.Second {
 			// TODO: should we get the standard logger here? if yes: How?
-			fmt.Printf("can't connect to nats (stan) server, retrying in %s\n", b.NextBackOff())
+			fmt.Printf("can't connect to nats (stan) server, retrying in %s\n", n)
```

Comment:
In order to address the above TODO item couldn't we just use 

```
logger.New().Error().Err(err).Msg("can't connect to nats (stan) server, retrying")
```

here? (https://github.com/cs3org/reva/blob/master/pkg/logger/logger.go#L48)
It's not exactly nice, but would at least be a bit cleaner than just doing fmt.Printf here.

Reply:
Not perfect. But much better than before :+1: